.travis.yml

```diff
@@ -43,7 +44,7 @@ env:
   - TARGET=SPRACINGF3
   - TARGET=SPRACINGF3EVO
 #  - TARGET=SPRACINGF3MINI
-  - TARGET=STM32F3DISCOVERY
```

Comment:
One of the reasons for the Travis builds is to pick up errors on submission of a PR. As @mikeller says, the STM32F3DISCOVERY target is useful for that since it has all build flags set. Can you add it back in please?

I also think it is questionable to include the BETAFLIGHTF3 board at this stage, since it is not yet available.